cookbooks/sysctl/recipes/tune_large_db.rb

```diff
@@ -0,0 +1,15 @@
+
+thp_filename = '/sys/kernel/mm/transparent_hugepage/enabled'
+if ::File.exists?(thp_filename)
+  execute 'disable transparent huge pages when present' do
+    command "echo never > #{thp_filename}"
+  end
+
+  sysctl "Adjust vm.dirty ratios for large instances" do
```

Comment:
Do we want to adjust these ratios regardless of the existence of /sys/kernel/mm/transparent_hugepage/enabled?